codegen/src/main/kotlin/komem/litmus/LitmusTestProcessor.kt

```diff
@@ -10,12 +10,13 @@ class LitmusTestProcessorProvider : SymbolProcessorProvider {
     }
 }
 
-class LitmusTestProcessor(val codeGenerator: CodeGenerator) : SymbolProcessor {
+class LitmusTestProcessor(private val codeGenerator: CodeGenerator) : SymbolProcessor {
```

Comment:
Do we still need this processor?

Reply:
All it does is 1) collect all tests from a package and 2) assign each test some name. We can manually put each new test into some single list, but that is very inconvenient (as I remember from the times before this processor came to be). Is it a problem to have it?

Reply:
Ok, if you like it, let's keep it.